full-service/src/json_rpc/json_rpc_request.rs

```diff
@@ -88,24 +89,27 @@ pub enum JsonCommandRequest {
         txo_id: String,
         output_values: Vec<String>,
         destination_subaddress_index: Option<String>,
-        fee: Option<String>,
+        fee_value: Option<String>,
+        fee_token_id: Option<String>,
```

Comment:
Why not `fee: Option<Amount>`?

Reply:
That's intended, if fee_token_id is omitted it will use the default, likewise for fee_value. Neither, one of, and both are valid states.